### PR TITLE
Feed integration [PART2] Create recycler view integration screen

### PR DESCRIPTION
**Pull request by OstapOpalynskyi-fw**
**What**
Create recycler view integration screen

**Why**
- ASDK-990

**How**
Copy existent code from the SDK V2

**Video**

https://user-images.githubusercontent.com/101661405/212875926-983d5ad5-db33-4b3e-894c-7352fd27678f.mp4

### CODE REVIEW

File: feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    abstract class BaseViewHolder(val view: View) : RecyclerView.ViewHolder(view) {
+        abstract fun bind(item: ListItem)
+    }
+
+    inner class FeedItemViewHolder(view: View) : BaseViewHolder(view) {
+        private var feedView: FwLifecycleAwareVideoFeedView? = null
+        override fun bind(item: ListItem) {
+            // This check is needed to avoid calling init() every time view goes in and out of the view port
+            if (feedView == null) {
+                val feedView: FwLifecycleAwareVideoFeedView = view.findViewById<FwLifecycleAwareVideoFeedView?>(R.id.feedView).apply {
```
Comment (mohsenoid) [inline feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListAdapter.kt:60]: `FwLifecycleAwareVideoFeedView`
HighFiveGIF

File: feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import tv.fw.feedintegration.BuildConfig.FW_PLAYLIST_ID
+import tv.fw.feedintegration.R
+import tv.fw.videofeed.FeedItemClickListener
+import tv.fw.videofeed.FeedViewState
+import tv.fw.videofeed.FeedViewStateListener
+import tv.fw.videofeed.baseOptions
+import tv.fw.videofeed.viewOptions
+
+/**
+ * A fragment representing a list of Items.
```
Comment (mohsenoid) [inline feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt:25]: Now I know where that came from https://github.com/loopsocial/firework_android_sdk_examples/pull/26#discussion_r1084041227 😁

File: feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        val playlistFeedViewOptions = viewOptions {
+            baseOptions {
+                supportBackwardAds(true)
+                feedResource(FeedResource.Playlist(FW_CHANNEL_ID, FW_PLAYLIST_ID))
+            }
+        }
+        return listOf(
+            TextItem(LOREN_IPSUM),
+            TextItem(LOREN_IPSUM),
+            FeedViewItem(UUID.randomUUID().toString(), playlistFeedViewOptions),
```
Comment (mohsenoid) [inline feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt:54]: I am wondering why not more feed views in the recycler view. 😈

**Review by mohsenoid — APPROVED** [review]
LGTM, minor comments

File: feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt (hunk 3 above)
Comment (OstapOpalynskyi-fw) [reply feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt:54]: Because there is issue with it

Comment (mohsenoid) [reply feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt:54]: Ah! do we have a bug ticket for it?

Comment (OstapOpalynskyi-fw) [reply feed_integration/app/src/main/java/tv/fw/feedintegration/recyclerview/FeedListFragment.kt:54]: It is tricky, and needs investigation
No ticket yet

**Comment by mohsenoid** [comment]
I fixed the CI issue

**Pull request merged**